python/cli/prelude_cli/views/iam.py

```diff
@@ -34,12 +35,20 @@ def register_account(controller):
               help='provide a mode',
               default=Mode.MANUAL.name, show_default=True,
               type=click.Choice([m.name for m in Mode], case_sensitive=False))
+@click.option('-c', '--company',
+              help='provide your associated company',
+              default='', show_default=True,
+              type=str)
 @click.pass_obj
 @handle_api_error
-def update_account(controller, mode):
+def update_account(controller, mode, company):
     """ Update an account """
     with Spinner():
-        controller.update_account(mode=Mode[mode.upper()].value)
+        data = controller.update_account(
+            mode=Mode[mode.upper()].value,
```

Comment:
watch out that both these flags are optional. you'll want to change the mode flag default to None and do an `if` check around it before sending it to the controller

we didn't used to need ^ when it the only thing you could update was mode, but now someone might want to update company without updating mode